cypress/e2e/tests/migration/migration-waves/sort.test.ts

```diff
@@ -93,6 +99,71 @@ describe(["@tier2"], "Migration Waves sort validations", function () {
         const afterDescSortList = getTableColumnData(endDate);
         verifyDateSortDesc(afterDescSortList, unsortedList);
     });
+    it("Sort Manage applications table", function () {
+        const applicationsList: Assessment[] = [];
+
+        const stakeholdersList = createMultipleStakeholders(3);
+        const businessServicesList = createMultipleBusinessServices(3);
+        for (let i = 0; i < 3; i++) {
+            let appdata = {
+                name: data.getRandomWord(4),
+                owner: stakeholdersList[i].name,
+                business: businessServicesList[i].name,
+            };
+            const application = new Assessment(appdata);
+            application.create();
+            applicationsList.push(application);
+        }
+        const { start: startDate, end: endDate } = generateRandomDateRange();
+        const migrationWave = new MigrationWave(data.getRandomWord(4), startDate, endDate);
+        migrationWave.create();
+        migrationWave.openManageApplications();
+        const unsortedappList = getTableColumnData("Application Name");
+        const unsortedbusinessList = getTableColumnData("Business service");
```

Comment:
`unsortedAppList` and `unsortedBusinessList` (App and Business start with uppercase)